microsoft-edge/extensions-chromium/getting-started/index.md

```diff
@@ -5,13 +5,13 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 11/07/2022
+ms.date: 02/07/2022
 ---
 # Extension concepts and architecture
 
 This article introduces concepts about building a Microsoft Edge extension.  Follow along to understand how multi-tab browsers work.
 
-A Microsoft Edge *extension* is a small app that adds or modifies features of Microsoft Edge.  For an introduction, see [Overview of Microsoft Edge extensions](../index.md).
+A Microsoft Edge *extension* is a small app that users may opt into that adds or modifies features of Microsoft Edge for a unique user experience.  For an introduction, see [Overview of Microsoft Edge extensions](../index.md).
```

Comment:
probably reword "for a unique user experience"; not sure the objective or value-add is to have a "unique user experience".

To draft another wording, I'd have to read the positioning phrases at
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/
Here's an existing phrasing from that Intro page that's pretty focused:
"An extension provides a niche experience or function that is important to a target audience."

I'd try wording like "specialized" -- I'd read the overview article and pick wording from there.

Reply:
So I added specialized to that phrase.  Later on there is the term unique identifier so it seems wise.